### PR TITLE
Fix `istioctl pc ep -f -` returns `Error: open -: no such file or directory`

### PR DESCRIPTION
**Pull request by kebe7jun**
**Please provide a description of this PR:**
Fix https://github.com/istio/istio/issues/43045

### CODE REVIEW

**Review by hanxiaop — APPROVED** [review]

File: releasenotes/notes/43045.yaml [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+# issue is a list of GitHub issues resolved in this note.
+# If issue is not in the current repo, specify its full URL instead.
+issue:
+  - 43045
+
+# releaseNotes is a markdown listing of any user facing changes. This will appear in the
+# release notes.
+releaseNotes:
+  - |-
+    **Fixed** `istioctl pc ep -f -` returns `Error: open -: no such file or directory`.
```
Comment (ericvn) [inline releasenotes/notes/43045.yaml:34]: I would expand out the command, proxy-config for example.

Comment (kebe7jun) [reply releasenotes/notes/43045.yaml]: fixed.

**Comment by kebe7jun** [comment]
@ericvn PTAL.

**Review by ericvn — APPROVED** [review]
Release notes look good.

**Pull request merged**